packages/manager/modules/hub/src/components/user-panel/shortcuts/shortcut-tile/template.html

```diff
@@ -0,0 +1,13 @@
+<div class="manager-hub-shortcut-tile">
+    <a data-ng-href="{{ $ctrl.url }}" class="manager-hub-shortcut-tile_icon">
+        <span class="oui-icon" data-ng-class="$ctrl.icon" aria-hidden="true">
```

Comment:
Doesn't the icon have a semantic meaning ? Maybe you should use `aria` to transmit this meaning then